CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.2.3
+  - Added info to dictionary_path description to explain why integers must be quoted
```

Comment:
Change this to:
```
## master
  - Added info to dictionary_path description to explain why integers must be quoted
```
We will do the final version change in the last PR, i.e. replace master with 3.2.3. Means you should revert the `gemspec` change too.